recipe/meta.yaml

```diff
@@ -13,15 +13,17 @@ source:
 build:
   number: 1
   string: cuda{{ cudatoolkit | replace(".*", "") }}_py{{ python | replace(".", "") }}_{{ PKG_BUILDNUM }}
+  script_env:
+    - CUDA_HOME
```

Comment:
As per Jason's PR https://github.com/open-ce/nvcc-feedstock/pull/5, this needs to be changed now to OPEN_CE_CUDA_HOME.

Reply:
and per https://github.com/open-ce/open-ce/pull/84 it should be back to CUDA_HOME if still needed at all